sites/page-details/includes/footer.html

```diff
@@ -1,27 +1,34 @@
 <footer class="pagedetails{% if include.addContainer %} container{% endif %}">
 	<h2 class="wb-inv">{{ i18nText-pageDetails }}</h2>
 
-	{%- if page.noReportProblem -%}
+	{% assign col-modified = "col-xs-12" %}
+
+	{%- if page.pageFeedback -%}
+		{%- if page.share -%}
+			{% assign col-share = "col-sm-4 col-md-3 col-sm-offset-1 col-md-offset-2 col-lg-offset-3" %}
+		{%- endif -%}
+	{%- elsif page.noReportProblem -%}
```

Comment:
Can the old RAP and the new OPUFT share the same grid configurations? Just asking, cause this will get tricky otherwise especially for implementors such as the AEM integration.

And my question goes both ways here: Can the OPUFT be adjusted to fit the previous RAP grid, or can the RAP gri fit in the new OPUFT grid?

OPUFT = On-Page User Feedback Tool, which is the new name for this component.

Reply:
It might be possible to modify the OPUFT to use as much width as possible (like what RAP does) and use an "inner" grid layout to keep its visual design intact. That should allow the share button's classes to be ``col-sm-4 col-md-3`` in both the OPUFT + RAP scenarios.

But keep in mind that if both OPUFT + RAP are disabled, the share button will still use a different set of classes (``col-sm-4 col-md-3 col-sm-push-8 col-md-push-9``) since it's placed beside date modified when the left widgets are disabled.

Not sure if this would be a show-stopper, but in order for the share button's top margin to only be activated when OPUFT is in play, the ``gc-pg-hlpfl`` class needs to be on an adjacent container. So it'd need to be on the same ``div`` element as the ``col-sm-8 col-md-9 col-lg-9`` classes.

@GormFrank Would keeping the ``gc-pg-hlpfl`` class at that spot be a problem?

Reply:
@EricDunsworth See comment above https://github.com/wet-boew/GCWeb/pull/1916#discussion_r995207738

Reply:
We're good to go on this front now :).

I managed to get OPUFT to fit within RAP's grid configuration as a part of https://github.com/wet-boew/GCWeb/pull/1916#discussion_r1012275546.

Btw I wasn't able to get OPUFT's grid widths to exactly match what I had before, but it's pretty close and no one's likely to notice the difference in practice.

Unfortunately this change will necessitate backporting a viewport-specific margin-right class from Bootstrap 4-5 to prevent the ``legend``'s grid from "sticking" to the yes/no button's grid. Probably ``me-md-2`` (aka ``mr-md-2`` in Bootstrap 4) or something similar.

Reply:
Added ``pr-sm-3`` (via wet-boew/wet-boew#9467) and ``pr-sm-3-nojs`` classes to  deal with the grid spacing.